### PR TITLE
[RHELC-997] Add a new rerun plan

### PR DESCRIPTION
**Pull request by danmyway**
* With recent changes to the nondestructive tests we have lost the possibility to run them individually
* Creating this plan should enable us to do so again
* The plan is created by concatenating all the .fmf files from the tier0 directory and performing neccessary modifications
* Add new pre-commit hook to regenerate the rerun.fmf file
* Simplify tags in the tier0 .fmf files

Jira Issues: [RHELC-997](https://issues.redhat.com/browse/RHELC-997)

Checklist

- [x] PR has been tested manually in a VM (either author or reviewer)
- [x] Jira issue has been made public if possible
- [x] `[RHELC-]` is part of the PR title 
- [ ] GitHub label has been added to help with Release notes 
- [x] PR title explains the change from the user's point of view
- [x] Code and tests are documented properly
- [x] The commits are squashed to as few commits as possible (without losing data) 
- [ ] When merged: Jira issue has been updated to `Release Pending` if relevant

### CODE REVIEW

**Comment by danmyway** [comment]
@SpyTec could you please take a look at the pre-commit hook?

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+        args:
+          - -c
+          - |
+            find tests/integration/tier0/ -name "*.fmf" -exec sh -c 'echo "/$(basename $(dirname {})| tr "-" "_"):"; sed -e "s/^/    /" {}' \; | awk '/^\// && NR>1{print ""; print ""}1'> plans/rerun.fmf
+            sed -i 'N; s/\(^[[:space:]]*\)tag+:\n\([[:space:]]*\)-[[:space:]]*\([^-[:space:]]*\)/\1discover+:\n\2filter: tag:\3/g; P; D' plans/rerun.fmf
+            sed -i '/^[[:space:]]*test: \|^[[:space:]]*pytest/d' plans/rerun.fmf
+            sed -i '/^[[:space:]]*tier: 0/d' plans/rerun.fmf
+            sed -i 's/\s\+$//' plans/rerun.fmf
+        language: system
+        stages: [ commit ]
```
Comment (SpyTec) [inline .pre-commit-config.yaml:24]: If it works, then LGTM!

**Review by r0x0d — APPROVED** [review]
Tested locally and it works just fine. 

I think that it should be good to run the hook one more time locally before merging, as it seems that there were some changes with the sub-man plans.

**Review by r0x0d — APPROVED** [review]

**Pull request merged**